src/content/docs/style-guide/images/embed-images.mdx

```diff
@@ -78,50 +131,44 @@ Descriptive captions help the reader know why the image matters. If it's a scree
 
 For more help with captions and other supporting text around images, see [Guidelines for explaining images](/docs/style-guide/writing-guidelines/screenshots-images#identify-images).
 
-## Add an inline image [#inline-images]
+## Add an inline image (MDX) [#inline-images-mdx]
 
-To use an inline image, add something like this:
+To use an inline image:
+* import the image at the top of the file
+* use the `img` tag in the file
+* add a class of `inline` to `img`
+* add any optional height/width in the `style` prop
 
 ```
-The UI includes a multiple app names indicator. ![Multiple app names indicator in New Relic One](./images/new-relic-distributed-tracing-multiapp-icon.png "Multiple apps name indicator in New Relic One")
+import exampleImage from 'images/example.png';
+
+The UI includes a multiple app names indicator <img src={exampleImage} class="inline" style={{height: "30px", width: "30px"}} alt="Multiple app names indicator in New Relic One" title="Multiple apps name indicator in New Relic One" />.
 ```
 
 If the inline image is being used as an icon, always describe it first. When you embed the icon image, follow the image with the word `icon` in the text. For example:
 
 ```
-Select your app's settings ![settings icon](./images/icon-settings.png "settings icon") icon.
-```
-
-## Add a fixed width, block level image [#fixed-width-img]
-
-Fixed width, block level images are similar in format to full column width images, except the original image width is smaller than the column width (800px) of a page. It's important that you edit the HTML like you would an inline image. This way the image will be rendered at 100% of the column width and also be responsive to smaller screen sizes.
-
-Use these images when a screenshot is a small part of the page with a width of less than 800px, but when it still needs a caption like a full width image.
+---
+title: Abbreviated example of a file
+---
 
-Here's an example of the HTML for a fixed width, block level image:
+import exampleImage from 'images/example.png';
 
-```
-<div style="width: 100%; max-width: Npx;">
-  <img alt="ALT TEXT" height="X" src="IMG_URL" title="FILENAME" width="N">
-</div>
-	
-<div class="dnd-legend-wrapper" style="width: 100%; max-width: Npx;">
-  <div class="meta"><p>CAPTION TEXT</p></div>
-</div>
+Select your app's settings <img src={exampleImage} class="inline" style={{height: "30px", width: "30px"}} alt="Multiple app names indicator in New Relic One" title="Multiple apps name indicator in New Relic One" /> icon.
 ```
 
 ## Icons [#icons]
 
-You can choose from a variety of icons to include in your docs: 
+You can choose from a variety of icons to include in your docs:
 
-* Feather icons (prefixed with `'fe-`), which replace our previous Font Awesome icons
-* New Relic icons (prefixed with `nr-`)
-* Logos for third-party products (prefixed with `logo-`)
+- Feather icons (prefixed with `'fe-`), which replace our previous Font Awesome icons
```

Comment:
Do we need to refer to Font Awesome anymore?

Reply:
🤔  probably not?